src/Components/GEOSldas_GridComp/GEOSlandpert_GridComp/LDAS_PertRoutines.F90

```diff
@@ -1928,7 +1912,6 @@ subroutine get_pert_select( N_pert_max, pert_grid_l, std_pert, &
     pert_select(1:N_pert_max) = 0
 
     do k=1,N_pert_max
-
        do i=1,pert_grid_l%N_lon
           do j=1,pert_grid_l%N_lat
 
```

Comment:
@weiyuan-jiang, thanks for adding the MPI_Allreduce, very elegant.   While I was looking at the code, I was wondering if we can/should eliminate the i and j loops here by using something like: 
`if (maxval( std_pert(k,:,:) ) > 0)  pert_select(k) = 1 `
Not sure if this executes faster than the double-loop, but it probably doesn't hurt.

Reply:
Yes, that looks better